src/main/java/Duke.java

```diff
@@ -1,3 +1,54 @@
+import java.util.Scanner;
+public class Duke {
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
```

Comment:
Can consider making the ___________________________ a constant to improve readibility and consistency